docker-compose-columbia.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
```

Comment:
I'm wondering, is it possible to put the docker-compose files in a folder instead of under the root directory?